spec/gitpoints/points_spec.rb

```diff
@@ -0,0 +1,9 @@
+require "spec_helper"
+
+describe GitPoints::UserPoints do
+  describe ".git_points" do
+    it "returns the points of the user" do
+      expect(GitPoints::UserPoints.new(git_points("test_user"))).to eq 50
```

Comment:
@hrs  hum so git_points is a class method, because in my rails app I want to say UserPoints.git_points("github_user_name") right? 

I still seam to have the scope issue